scripts/CSSRE/lib/py/kafka.py

```diff
@@ -0,0 +1,86 @@
+import openshift as oc
```

Comment:
`refactor`: I would put this in the `Script` class so it'll be available in every script.